test/feature/AsyncGenerators/resources/observable.js

```diff
@@ -0,0 +1,71 @@
+const generator_ = Symbol();
```

Comment:
Even in this case? Before symbols, I would write: `this.generator_`. Now I would have written: `this[generator_]` to make the property truely private.

If there was an accessor by the same name, I would get a naming problem if the symbol of the private property was also named without an underscore, wouldn't I?